### ISSUE TITLE
Redfish schema allows special chars in "title" field.

### ISSUE THREAD

**Issue by hramasub** [cwacek/python-jsonschema-objects#50]
The schema which models a computer system collection (http://redfish.dmtf.org/schemas/v1/ComputerSystemCollection.json) allows for special characters (# and .) in the **title**field.

Does python-jsonschema-objects allow/or support **special chars in title field** ?

The above schema has the following title field:
  "title": "**#**ComputerSystemCollection**.**ComputerSystemCollection",

Here is the code snippet in which I **removed** the special chars for referring the namespace object:

with open('**ComputerSystemCollection.json**') as schema_file:
    root_schema = json.load(schema_file)

builder = pjso.ObjectBuilder (root_schema)
ns = builder.build_classes()
csc = **ns.ComputerSystemCollectionComputerSystemCollection**

Is this the correct access syntax ?

**Comment by cwacek** [comment]
Special characters are for obvious reasons a bit awkward. However, you should be able to access the generated class using Dictionary access as follows: `ns['#ComputerSystemCollection.ComputerSystemCollection']`

Did you try that?

**Comment by hramasub** [comment]
I tried that now though I am still blocked by the error:
NotImplementedError: anyOf is not supported as bare property

I had raised a separate issue (https://github.com/cwacek/python-jsonschema-objects/issues/51) for that.

In essence, until build_classes() succeeds,  I have no way of validating the access syntax which you suggested.

Thanks and much appreciate your response.

**Comment by cwacek** [comment]
The exact syntax to use is `ns.classes['#ComputerSystemCollection.ComputerSystemCollection']`, but this works.
